lib/bigquery/dataset.js

```diff
@@ -165,18 +185,8 @@ Dataset.prototype.delete = function(options, callback) {
  * @example
  * dataset.getMetadata(function(err, metadata, apiResponse) {});
  */
-Dataset.prototype.getMetadata = function(callback) {
-  var that = this;
-  this.makeReq_('GET', '', null, null, function(err, resp) {
-    if (err) {
-      callback(err, null, resp);
-      return;
-    }
-
-    that.metadata = resp;
-
-    callback(null, that.metadata, resp);
-  });
+Dataset.prototype.getMetadata = function() {
+  ServiceObject.prototype.getMetadata.apply(this, arguments);
```

Comment:
It took me a minute to remember why it's like this way (documentation purposes, yes?) . I believe it's possible to document methods/class/etc. without the code actually being present. Although I'm not sure what kind of effort that would involve to bring into our current docs site.